charts/tekton-apps/values-test.yaml

```diff
@@ -29,6 +29,9 @@ apps:
         pm: VadikKuznetsov
         tm: AlekseyBashirov
       namespace: staging
+      notifications:
```

Comment:
Please update also docs with these changes, currently we try to describe all possible features of `tekton-apps` app in docs.